winterm/ansi.go

```diff
@@ -164,10 +165,13 @@ func GetStdFile(nFile int) (*os.File, uintptr) {
 	var file *os.File
 	switch nFile {
 	case syscall.STD_INPUT_HANDLE:
+	case windows.STD_INPUT_HANDLE:
 		file = os.Stdin
 	case syscall.STD_OUTPUT_HANDLE:
+	case windows.STD_OUTPUT_HANDLE:
 		file = os.Stdout
 	case syscall.STD_ERROR_HANDLE:
+	case windows.STD_ERROR_HANDLE:
 		file = os.Stderr
```

Comment:
I don't think this will work as expected as golang either needs an explicit `fallthrough`, or "same options" have to be combined in a single `case` as comma-separated values;

https://play.golang.org/p/ytjHf-Bdxum

```go
package main

import (
	"fmt"
)

func main() {
	for _, v := range []int{1, 2, 3, 4, 5, 6, 7} {
		result := "unknown"
		switch v {
		case 1: // doesn't fall-through, so this is an empty "case"
		case 2:
			result = "found one or two"
		case 3:
			fallthrough // falls through to "case 4"
		case 4:
			result = "found three or four"
		case 5, 6: // matches either 5 or 6
			result = "found five or six"
		default:
			result = "found something else"
		}
		fmt.Printf("%d: %s\n", v, result)
	}
}
```

The above will print:

```
1: unknown
2: found one or two
3: found three or four
4: found three or four
5: found five or six
6: found five or six
7: found something else
```

Let me open a PR to fix

Reply:
Opened https://github.com/Azure/go-ansiterm/pull/32